worker/charts/worker/values.yaml

```diff
@@ -31,14 +31,14 @@ resources:
     cpu: "100m"
     memory: "128Mi"
   limits:
-    cpu: "300m"
-    memory: "384Mi"
+    cpu: "500m"
+    memory: "512Mi"
 
 autoscaling:
-  enabled: true
+  enabled: false
```

Comment:
users should have to opt in to autoscaling as this may be unexpected for basic deployments.